windows/windows-api/src/main/java/com/duckduckgo/windows.api/WindowsSettingsNav.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2023 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.windows.api
+
+import android.content.Context
+import android.content.Intent
+
+/**
+ * Public interface to provide navigation Intents related to Windows App Settings
+ */
+interface WindowsSettingsNav {
+    fun openWindowsSettings(context: Context): Intent
```

Comment:
I would rename this to `activityContext` and even probably annotated with `ActivityContext`. That's the only reason you need a context here, if it was the app `context` you would not need it as a parameter

Reply:
Sounds good 👍